pkg/vpcmodel/commonConnectivity.go

```diff
@@ -6,7 +6,114 @@ SPDX-License-Identifier: Apache-2.0
 
 package vpcmodel
 
-import "github.com/np-guard/models/pkg/connection"
+import (
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/netp"
+)
+
+// todo: remove stateful from connection.Set (for both options)
+
+// ExtendedSet connection details
+type ExtendedSet struct {
```

Comment:
consider rename "ExtendedSet"
its more than a Set..

Reply:
renamed to SetWithStateful

Reply:
its not a set, the name should be something that related to connections

Reply:
`ConnWithStateful`